src/Polly.Core/Simmy/Latency/LatencyStrategyOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Polly.Simmy.Latency;
+
+#pragma warning disable CS8618 // Required members are not initialized in constructor since this is a DTO, default value is null
+
+/// <summary>
+/// Represents the options for the Latency chaos strategy.
+/// </summary>
+public class LatencyStrategyOptions : MonkeyStrategyOptions
+{
+    internal static readonly TimeSpan DefaultLatency = TimeSpan.FromSeconds(30);
+
+    /// <summary>
+    /// Gets or sets the delegate that's raised when delay occurs.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see langword="null"/>.
```

Comment:
Should we document what that signifies?

Reply:
I'd say it's clear but feel free to make a suggestion 😀